python-lib/dku_timeseries/resampling.py

```diff
@@ -94,8 +99,8 @@ def transform(self, df, datetime_column, groupby_columns=None):
         # when having multiple timeseries, their time range is not necessarily the same
         # we thus compute a unified time index for all partitions
         reference_time_index = self._compute_full_time_index(df_copy, datetime_column)
-        columns_to_resample = [col for col in df_copy.select_dtypes([int, float]).columns.tolist() if col != datetime_column and col not in groupby_columns]
-        category_columns = [col for col in df.select_dtypes([object, bool]).columns.tolist() if col != datetime_column and col not in columns_to_resample and
+        columns_to_resample = [col for col in df_copy.select_dtypes([int, float, np.float32, np.int32]).columns.tolist() if col != datetime_column and col not in groupby_columns]
```

Comment:
very nitpicky but this could be an array somewhere to avoid forgetting to update both type list in the future :)